src/Android/Properties/AndroidManifest.xml

```diff
@@ -1,80 +1,46 @@
-  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
-  <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
-  <uses-permission android:name="com.samsung.android.providers.context.permission.WRITE_USE_APP_FEATURE_SURVEY" />
-
-  <uses-feature android:name="android.hardware.camera" android:required="false" />
-  <uses-feature android:name="android.hardware.camera.autofocus" android:required="false" />
-
-  <application
-    android:label="Bitwarden"
-    android:theme="@style/LaunchTheme"
-    android:allowBackup="false"
-    tools:replace="android:allowBackup"
-    android:icon="@mipmap/ic_launcher"
-    android:roundIcon="@mipmap/ic_launcher_round"
-    android:networkSecurityConfig="@xml/network_security_config">
-    <provider
-      android:name="androidx.core.content.FileProvider"
-      android:authorities="com.x8bit.bitwarden.fileprovider"
-      android:exported="false"
-      android:grantUriPermissions="true">
-      <meta-data
-        android:name="android.support.FILE_PROVIDER_PATHS"
-        android:resource="@xml/filepaths" />
-    </provider>
-
-    <receiver
-      android:name="com.google.firebase.iid.FirebaseInstanceIdInternalReceiver"
-      android:exported="false" />
-    <receiver
-      android:name="com.google.firebase.iid.FirebaseInstanceIdReceiver"
-      android:exported="true"
-      android:permission="com.google.android.c2dm.permission.SEND">
-      <intent-filter>
-        <action android:name="com.google.android.c2dm.intent.RECEIVE" />
-        <action android:name="com.google.android.c2dm.intent.REGISTRATION" />
-        <category android:name="com.x8bit.bitwarden" />
-      </intent-filter>
-    </receiver>
-
-    <meta-data android:name="android.max_aspect" android:value="2.1" />
-    <meta-data android:name="android.content.APP_RESTRICTIONS" android:resource="@xml/app_restrictions" />
-
-    <!-- Support for Samsung "Multi Window" mode (for Android < 7.0 users) -->
-    <meta-data android:name="com.samsung.android.sdk.multiwindow.enable" android:value="true" />
-    <meta-data android:name="com.samsung.android.sdk.multiwindow.penwindow.enable" android:value="true" />
-
-    <!-- Support for LG "Dual Window" mode (for Android < 7.0 users) -->
-    <meta-data android:name="com.lge.support.SPLIT_WINDOW" android:value="true" />
-  </application>
-
-  <!-- Package visibility (for Android 11+) -->
-  <queries>
-      <intent>
-          <action android:name="android.intent.action.VIEW"/>
-          <data android:scheme="http"/>
-      </intent>
-      <intent>
-          <action android:name="android.intent.action.VIEW"/>
-          <data android:scheme="https"/>
-      </intent>
-  </queries>
-
-</manifest>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools" android:versionCode="1" android:versionName="2.9.2" android:installLocation="internalOnly" package="com.x8bit.bitwarden">
```

Comment:
Need to revert this file (VSmac refuses to leave it alone)

Reply:
maaaaan I forgot about this